types/setting.go

```diff
@@ -858,6 +861,23 @@ var (
 		ReadOnly: false,
 		Default:  "false",
 	}
+
+	SettingDefinitionUnmapMarkSnapChainRemoved = SettingDefinition{
+		DisplayName: "Unmap Mark Snapshot Chain as Removed",
```

Comment:
This determines the extra operations during the filesystem trim rather than if the trim is allowed. How about `Remove Snapshots during Filesystem Trim`?